bookprice_sale_account/models/account_move_line.py

```diff
@@ -0,0 +1,26 @@
+from odoo import api, fields, models
+
+
+class AccountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    # computational field for book price    
+    book_price = fields.Float(
+        string="Book Price",
+        compute="_compute_book_price",
+        help="Price without any discount and taxes",
+    )
+
+    @api.depends('product_id', 'quantity', 'move_id.partner_id.property_product_pricelist')
+    def _compute_book_price(self):
+        # Compute the book price for the invoice line for specified product, quantity and unit of measure.
+        for line in self:
+            if line.product_id and line.move_id and line.move_id.invoice_line_ids:
+                line.book_price = line.move_id.partner_id.specific_property_product_pricelist._get_product_price(
```

Comment:
```suggestion
                line.book_price = line.order_id.pricelist_id._get_product_price(
```
why not using pricelist on sale order? is there any case I'm missing?